src/WebJobs.Script.WebHost/Properties/Resources.resx

```diff
@@ -259,4 +259,28 @@
     ]
 }</value>
   </data>
+  <data name="TraceAddOrUpdateFunctionSecret" xml:space="preserve">
+    <value>{0} secret '{1}' for '{2}' {3}.</value>
+  </data>
+  <data name="TraceFunctionSecretGeneration" xml:space="preserve">
+    <value>Function ('{0}') secrets do not exist. Generating secrets.</value>
+  </data>
+  <data name="TraceHostSecretGeneration" xml:space="preserve">
+    <value>Host secrets do not exist. Generating secrets.</value>
+  </data>
+  <data name="TraceKeysApiSecretChange" xml:space="preserve">
+    <value>Secret '{0}' for '{1}' {2}.</value>
```

Comment:
Might be able to consolidate with fomat expression as TraceAddOrUpdateFunctionSecret above?